jwst/datamodels/schemas/dark.schema.yaml

```diff
@@ -1,6 +1,9 @@
 title: Dark current data model
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: keyword_filter.schema.yaml
+- $ref: keyword_exptype.schema.yaml
```

Comment:
Wait a sec. I just now dawns on me: why are we including FILTER and EXPTYPE keyword definitions in the dark data model schema? Darks are selected on DETECTOR, SUBARRAY, and sometimes READPATT. They are never selected based on either FILTER or EXPTYPE. So we don't need FILTER or EXPTYPE keywords in darks and we DO need READPATT (at least in some of them). I assume DETECTOR is already included in the standard list contained in the new referencefile.schema.yaml.